sql_runner.go

```diff
@@ -141,6 +141,9 @@ func (sr *sqlRunner) runSQL(db database, key string) bool {
 	scanner := bufio.NewScanner(stdout)
 	for scanner.Scan() {
 		line := scanner.Text()
+		if sr.typ == postgreSQL {
+			line = strings.Replace(line, " | ", "\t", -1)
```

Comment:
This is dodgy as :trollface: 
All you need is to add this to the CLI call:

```
-tAF '        '
```

That's a literal tab inside the quotes.